deepchecks/utils/distribution/drift.py

```diff
@@ -24,9 +24,79 @@
     preprocess_2_cat_cols_to_same_bins
 from deepchecks.utils.strings import format_percent
 
+__all__ = ['calc_drift_and_plot', 'get_drift_method']
+
+
 PSI_MIN_PERCENTAGE = 0.01
+SUPPORTED_CATEGORICAL_METHODS = ['Cramer\'s V', 'PSI']
+SUPPORTED_NUMERIC_METHODS = ['Earth Mover\'s Distance']
+
+
+def get_drift_method(result_dict: Dict):
+    """Return which drift scoring methods were in use.
+
+    Parameters
+    ----------
+    result_dict : Dict
+        the result dict of the drift check.
+    Returns
+    -------
+    Tuple(str, str)
+        the categorical scoring method and then the numeric scoring method.
+
+    """
+    result_df = pd.DataFrame(result_dict).T
+    cat_mthod_arr = result_df[result_df['Method'].isin(SUPPORTED_CATEGORICAL_METHODS)]['Method']
+    if len(cat_mthod_arr):
+        cat_method = cat_mthod_arr[0]
+    else:
+        cat_method = None
```

Comment:
possible to reduce to one-liner
```suggestion
   cat_method = cat_mthod_arr[0] if cat_mthod_arr else None
```